src/Schema/BulkLoader/ExtensionLoader.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace SilverStripe\GraphQL\Schema\BulkLoader;
+
+use SilverStripe\Core\Extensible;
+use InvalidArgumentException;
+use SilverStripe\Core\Extension;
+
+/**
+ * Loads classes that have a given extension assigned to them.
+ */
+class ExtensionLoader extends AbstractBulkLoader
+{
+    const IDENTIFIER = 'extensionLoader';
```

Comment:
```suggestion
    public const IDENTIFIER = 'extensionLoader';
```
Also update it for all subsequent IDENTIFIER's in following files